drivers/hmis/spec/factories/hmis/hud/clients.rb

```diff
@@ -45,8 +45,14 @@
       with_custom_client_name { false }
     end
     after(:build) do |client, evaluator|
-      HudUtility2024.races.except('RaceNone').keys.each { |f| client.send("#{f}=", [0, 1].sample) }
-      HudUtility2024.gender_fields.excluding(:GenderNone).each { |f| client.send("#{f}=", [0, 1].sample) }
+      race_attributes = HudUtility2024.races.except('RaceNone').keys.map { |r| [r, [0, 1].sample] }.to_h
+      race_attributes['RaceNone'] = 99 if race_attributes.values.sum.zero?
+      client.assign_attributes(race_attributes)
```

Comment:
just updating factory to always set valid race attributes. (if all races are 0, RaceNone should be non-null)